src/network_models/power_flow_evaluation.jl

```diff
@@ -385,6 +390,8 @@ _get_pf_result(::Type{PowerFlowLineActivePowerFromTo}, pf_data::PFS.PowerFlowDat
     PFS.get_branch_activepower_flow_from_to(pf_data)
 _get_pf_result(::Type{PowerFlowLineActivePowerToFrom}, pf_data::PFS.PowerFlowData) =
     PFS.get_branch_activepower_flow_to_from(pf_data)
+_get_pf_result(::Type{PowerFlowLossFactors}, pf_data::PFS.PowerFlowData) =
+    isnothing(pf_data.loss_factors) ? nothing : pf_data.loss_factors
```

Comment:
wait doesn't this just simplify to
```suggestion
    pf_data.loss_factors
```
?

Reply:
this should be fine because of the change to only include the PowerFlowLossFactors in the list if it needs to be there and will not be nothing